skiko/src/awtMain/objectiveC/macos/MetalRedrawer.mm

```diff
@@ -133,6 +133,10 @@ JNIEXPORT jlong JNICALL Java_org_jetbrains_skiko_redrawer_MetalRedrawer_createMe
         [container setNeedsDisplayOnBoundsChange: YES];
 
         AWTMetalLayer *layer = [AWTMetalLayer new];
+        if (frameBuffering == 2 || frameBuffering == 3) {
```

Comment:
We either need to write it unconditionally, or add `else` branch. Otherwise setting frameBuffering = 2, then 0 won't change anything